saleor/graphql/shop/tests/queries/test_shop.py

```diff
@@ -830,3 +830,80 @@ def test_query_countries_filter_shiping_zones_attached_none(
 
     # then
     assert len(data) == len(countries)
+
+
+SHOP_METADATA_QUERY = """
```

Comment:
Could you move does tests to `saleor/graphql/meta/tests/queries/` directory? You should create there a new `test_shop.py` file and move those tests there :)

Reply:
And we should also add tests for mutations in `saleor/graphql/meta/tests/mutations/` directory.